oceanwaves/spectral.py

```diff
@@ -4,6 +4,8 @@
 
 from oceanwaves.utils import *
 
+from warnings import warn
```

Comment:
We use the *logging* module in this package, which can also be used for warnings:

```
import logging
logger = logging.getLogger(__name__)
logger.warn('Frequency array contains zeros.')
```

See for example *oceanwaves.py*.